edit-post/index.js

```diff
@@ -1,6 +1,9 @@
 /**
  * WordPress dependencies
  */
+import '@wordpress/core-data';
+import '@wordpress/editor';
+import '@wordpress/nux';
```

Comment:
viewport?

Reply:
There is no explicit usage in the module so I missed it ...
I will fix.

We need to figure a more robust way 😄

Reply:
>There is no explicit usage in the module so I missed it ...

This confused me a bit. If there's not explicit usage, the module shouldn't need to define it?

But in looking closer, there is explicit usage:

https://github.com/WordPress/gutenberg/blob/d7eb4f697e9685501e765a25f75816875457e887/edit-post/store/effects.js#L142

Reply:
It was all my fault, I didn't grep properly ...